README.md

```diff
@@ -1,2 +1,14 @@
-## Acknowledgments
+## Setup
+
+ - Install unity editor version `6000.0.63f1` (LTS)
+ - Download the Tobii game development API. This code was built and tested with version 9.0.4.26.
+ - You need to add `.net/Tobii.GameIntegration.Net.dll` and all the `.dll` and `.lib` files in the `lib` folder to your project.
+
+You may need to [download the drivers](gaming.tobii.com/getstarted/) for the Tobii Eyetracker 5, and calibrate the tracker using the Tobii Experience app.
+
+## Acknowledgements
+
+Rocket sprites by [JM. Atencia](https://opengameart.org/users/jmatencia) [CC-BY 3.0](http://creativecommons.org/licenses/by/3.0/)
```

Comment:
It looks to me like you didn't use the that rocket, so could delete the acknowlegement?